pkg/controller/podautoscaler/horizontal.go

```diff
@@ -1177,14 +1181,14 @@ func setConditionInList(inputList []autoscalingv2.HorizontalPodAutoscalerConditi
 	return resList
 }
 
-func max(a, b int32) int32 {
+func maxChange(a, b int32) int32 {
```

Comment:
Yeah, you comment  confused me. ^_^
In order to avoid further ambiguity, **double confirm**, which one you tend to:
1. first
```
const (
	// MaxChangePolicySelect selects the policy with the highest possible change.
	MaxChangePolicySelect ScalingPolicySelect = "Max"
	// MinPolicySelect selects the policy with the lowest possible change.
	MinChangeChangePolicySelect ScalingPolicySelect = "Min"
	// DisabledPolicySelect disables the scaling in this direction.
	DisabledPolicySelect ScalingPolicySelect = "Disabled"
)
```
2. second
```
const (
	// MaxPolicySelect selects the policy with the highest possible change.
	MaxPolicySelect ScalingPolicySelect = "MaxChange"
	// MinPolicySelect selects the policy with the lowest possible change.
	MinPolicySelect ScalingPolicySelect = "MinChange"
	// DisabledPolicySelect disables the scaling in this direction.
	DisabledPolicySelect ScalingPolicySelect = "Disabled"
)
```
3. third
```
const (
	// MaxChangePolicySelect selects the policy with the highest possible change.
	MaxChangePolicySelect ScalingPolicySelect = "MaxChange"
	// MinChangePolicySelect selects the policy with the lowest possible change.
	MinChangePolicySelect ScalingPolicySelect = "MinChange"
	// DisabledPolicySelect disables the scaling in this direction.
	DisabledPolicySelect ScalingPolicySelect = "Disabled"
)
```
I prefer the **first one** because it doesn’t change how users use it